canonical/ubuntu.com
# Cred customer purchasing

## Done

- A page for buying activation keys for CUE0.1 Exam

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- Visit /credentials/shop
  - Select quantity
  - Pay
  - New activation keys should be visible in /credentials/shop/manage

## Issue / Card
Fixes [#WD-3821](https://warthogs.atlassian.net/browse/WD-3821)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
